### PR TITLE
CORE-11 Import coerce-tool-import-requests middleware

### PR DESCRIPTION
**Pull request by psarando**
This PR will import `apps.routes.schemas.tool/coerce-tool-import-requests` and `coerce-tool-list-import-request` into `common-swagger-api.schema.tools`, along with related functions into `common-swagger-api.schema.containers` and `common-swagger-api.coerce`.

Missed in #23.

### CODE REVIEW

File: src/common_swagger_api/schema/containers.clj [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
-  (:require [schema.core :as s]))
+  (:require [common-swagger-api.coerce :as coerce]
+            [schema.core :as s]))
+
+(defn coerce-settings-long-values
+  "Converts any values in the given settings map that should be a Long, according to the Settings schema."
+  [settings]
+  (-> settings
+      (coerce/string->long :memory_limit)
+      (coerce/string->long :min_memory_limit)
```
Comment (ash3rz) [inline src/common_swagger_api/schema/containers.clj:11]: (Not familiar with the code, but) The UI sends a `memory_limit` key, I'm guessing that gets changed to `min_memory_limit` somewhere?

Comment (ash3rz) [reply src/common_swagger_api/schema/containers.clj:11]: Oh wait, actually the UI's `ToolContainer` model has both keys... ignore my comment, I've gotten less familiar with Tools, I'm probably mixing things up 😅

Comment (psarando) [reply src/common_swagger_api/schema/containers.clj:11]: No problem. These are actually separate settings. According to the API docs:
```
min_memory_limit (integer, optional): The minimum about of memory (in bytes) that is required to run the tool container ,
memory_limit (integer, optional): The amount of memory (in bytes) that the tool container is restricted to ,
```
So `memory_limit` is a maximum limit. The UI can submit a value lower than the max allowed by the service, but the service will cap the limit for anything higher in the request.

I don't think the `min_memory_limit` setting is in use yet, but the service is storing it in the database for now.

**Review by slr71 — APPROVED** [review]
LGTM :shipit:

**Comment by psarando** [comment]
Thanks for the reviews! 👍

**Pull request merged**